### PR TITLE
Add documentation about the quota information functions

### PR DESCRIPTION
**Pull request by jgoizueta**
See #331

### CODE REVIEW

**Comment by jgoizueta** [comment]
cc @ethervoid

File: doc/quota_information.md [hunk 1]
```diff
@@ -12,3 +12,133 @@ Quota consumption is calculated based on the number of request made for each fun
...
+
+#### Returns
+
+The result is a *boolean* value. A *true* value (`'t'`) indicates that the available quota
+for the service is enough for the input size requested. A *false* value (`'f'`) indicates
+insufficient quota.
+
+#### Example
+
+Imagine you wish to geocode a table named
```
Comment (rafatower) [inline doc/quota_information.md]: named ..?

File: doc/quota_information.md [hunk 2]
```diff
@@ -12,3 +12,133 @@ Quota consumption is calculated based on the number of request made for each fun
...
+#### Returns
+
+The result is a *boolean* value. A *true* value (`'t'`) indicates that the available quota
+for the service is enough for the input size requested. A *false* value (`'f'`) indicates
+insufficient quota.
+
+#### Example
+
+Imagine you wish to geocode a table named 
+To check if you have enogh quota is available and avoiding ending with a partially
```
Comment (rafatower) [inline doc/quota_information.md]: *In order to check that you have enough quota and avoid a "quota exhausted" exception...*

File: doc/quota_information.md [hunk 3]
```diff
@@ -12,3 +12,133 @@ Quota consumption is calculated based on the number of request made for each fun
...
+This function is useful to check if enough quota is available for completing a job.
+
+This is specially relevant if a number of service calls are to be performed inside a transaction:
+if any of the calls fails due to exceeded quota the transaction will be rolled back, resulting
+in some quota being consumed but no saved results from the services consumed.
+
+In the case of of calling repeatedly quota-consuming functions (e.g. to geocode a whole table) it is 
+extremely important to first check if enough quota is available to complete the job using
+this function.
+
```
Comment (rafatower) [inline doc/quota_information.md]: I'd add the following:

Mind that some services consume more than one credit per row/call. E.g: isolines with more than one range/track would consume `N rows x M ranges` credits and therefore the input size should be `N x M`.

**Comment by rafatower** [comment]
After addressing the comments we should put Christa in the loop.

File: doc/quota_information.md (hunk 3 above)
Comment (jgoizueta) [inline doc/quota_information.md]: good point

File: doc/quota_information.md (hunk 1 above)
Comment (jgoizueta) [inline doc/quota_information.md]: 😝

File: doc/quota_information.md (hunk 2 above)
Comment (jgoizueta) [inline doc/quota_information.md]: Much better

**Comment by rafatower** [comment]
@csobier can you please review?

**Comment by csobier** [comment]
@rafatower, I just applied some minor grammar rewrites and fixed some styling of headings, etc. Otherwise, looks good!

File: doc/quota_information.md [hunk 4]
```diff
@@ -101,9 +94,7 @@ insufficient quota.
 
 #### Example
 
-Imagine you wish to geocode a whole table.
-In order to check that you have enough quota and avoid a "quota exhausted" exception
-you should first find out how many records you need to geocode:
+Suppose you want to geocode a whole table. In order to check that you have enough quota, and avoid a "quota exhausted" exceptio, first find out how many records you need to geocode:
```
Comment (rafatower) [inline doc/quota_information.md]: note to myself: missing n in "exceptio"

**Comment by rafatower** [comment]
Note: this PR contains just documentation. Tests are broken in mater and development branches, most likely because of a temporal condition. https://github.com/CartoDB/dataservices-api/issues/335

**Pull request merged**